docs/validators/advanced-setups.mdx

```diff
@@ -0,0 +1,102 @@
+---
+description: Using Connect with Advanced Validator Infrastructure
+title: Advanced Setups
+icon: plug
+---
+
+## Using Remote Signers <Icon icon="key-skeleton-left-right" />
+
+Remote signers can be used with Connect, however only certain versions are compatible.
+
+### Horcrux
+
+#### Required Version
+
+v3.3.0+
```

Comment:
Maybe we should link to releases?

Reply:
[link to releases](https://github.com/skip-mev/slinky/pull/604/commits/6a3bbde890b8f4b42a09a0c0587018375204ff22)